dev/bots/test/analyze_test.dart

```diff
@@ -54,51 +54,42 @@ void main() {
-            '║ test/analyze-test-input/root/packages/foo/deprecation.dart:72: Deprecation notice does not accurately indicate a beta branch version number; please see RELEASES_URL to find the latest beta build version number.',
-            '║ test/analyze-test-input/root/packages/foo/deprecation.dart:78: Deprecation notice does not accurately indicate a beta branch version number; please see RELEASES_URL to find the latest beta build version number.',
-            '║ test/analyze-test-input/root/packages/foo/deprecation.dart:101: Deprecation notice does not match required pattern. You might have used double quotes (") for the string instead of single quotes (\').',
-          ]
-          .map((String line) {
-            return line
-                .replaceAll('/', Platform.isWindows ? r'\' : '/')
-                .replaceAll(
-                  'STYLE_GUIDE_URL',
-                  'https://github.com/flutter/flutter/blob/main/docs/contributing/Style-guide-for-Flutter-repo.md',
-                )
-                .replaceAll(
-                  'RELEASES_URL',
-                  'https://flutter.dev/docs/development/tools/sdk/releases',
-                );
-          })
-          .join('\n');
-      expect(
-        result,
-        '╔═╡ERROR #1╞════════════════════════════════════════════════════════════════════\n'
-        '$lines\n'
-        '║ See: https://github.com/flutter/flutter/blob/main/docs/contributing/Tree-hygiene.md#handling-breaking-changes\n'
-        '╚═══════════════════════════════════════════════════════════════════════════════\n',
-      );
-    },
-    // TODO(goderbauer): Update and re-enable this after formatting changes have landed.
-    skip: true,
-  );
+  test('analyze.dart - verifyDeprecations', () async {
+    final String result = await capture(
+      () => verifyDeprecations(testRootPath, minimumMatches: 2),
+      shouldHaveErrors: true,
+    );
+    final String lines = <String>[
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:14: Deprecation notice should be a grammatically correct sentence and start with a capital letter; see style guide: STYLE_GUIDE_URL',
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:20: Deprecation notice should be a grammatically correct sentence and end with a period; notice appears to be "Also bad grammar".',
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:25: Deprecation notice must be an adjacent string.',
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:28: Deprecation notice must be an adjacent string.',
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:33: Deprecation notice must be an adjacent string.',
+          '║ test/analyze-test-input/root/packages/foo/deprecation.dart:52: Deprecation notice does not accurately indicate a beta branch version number; please see RELEASES_URL to find the latest beta build version number.',
```

Comment:
Surprised that `test8` (line 44) doesn't produce a warning here. According to `analyze-test-input/root/packages/foo
/deprecation.dart` it "should fail".

Reply:
I check this on an older commit that is a couple month old:

https://github.com/flutter/flutter/blob/d755bc222b2861146d64cf0570c4ba37db970918/dev/bots/test/analyze_test.dart
https://github.com/flutter/flutter/blob/d755bc222b2861146d64cf0570c4ba37db970918/dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart

Reply:
From https://github.com/flutter/flutter/blob/d755bc222b2861146d64cf0570c4ba37db970918/dev/bots/analyze.dart#L635-L639

From the code I think the test case is just mislabeled and it should succeed?

Reply:
I see. Thanks for checking. Can you correct the label of the test case to indicate that it should pass?